firmware/src/board/ublox_msg.hpp

```diff
@@ -128,9 +128,14 @@ struct CFG_NAV5
         Sea         = 5,
         Airborne_1g = 6,
         Airborne_2g = 7,
-        Airborne_4g = 8
+        Airborne_4g = 8,
+        Wrist       = 9 // only exsits in proto >= 18
     } dynModel;
 
+#define GNSS_DYN_MODEL_MIN     0
+#define GNSS_DYN_MODEL_MAX     9
+#define GNSS_DYN_MODEL_DEFAULT 8
```

Comment:
Macros can only be used for purposes of conditional compilation, and they are never allowed within the namespace scope. Here please replace that with this:

```c++
static constexpr U1 DynModelMin = 0;
static constexpr U1 DynModelMax = 8;
```

The default should not be declared in the driver, this violates the separation of concerns. Please move it to the definition of the configuration parameter.